docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Getting started with Dagster Cloud using a Hybrid deployment | Dagster Cloud
+
+platform_type: "cloud"
+---
+
+# Getting Started with Dagster Cloud using a Hybrid deployment
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud using a [Hybrid deployment](/dagster-cloud/deployment/hybrid). With a Hybrid deployment, your Dagster code is executed in your infrastructure. Refer to the [Getting started with Serverless deployment guide](/dagster-cloud/getting-started/getting-started-with-serverless-deployment) if you want to use Serverless.
+
+We'll walk you through setting up your Dagster Cloud account, setting up an agent, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- **Access to some Dagster code.** You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
```

Comment:
- **The `dagster` and `dagster-cloud` CLI installed.** These can be installed by running `pip install dagster dagster-cloud`